_aggregations/metric/extended-stats.md

```diff
@@ -26,7 +59,9 @@ GET opensearch_dashboards_sample_data_ecommerce/_search
 ```
 {% include copy-curl.html %}
 
-#### Example response
+## Example response
+
+The response is as follows:
```

Comment:
Can we add a little more context here? Something like "The response contains extended stats for `taxful_total_price`:"